frontend/src/metabase/embedding-sdk/types/components/data-picker.ts

```diff
@@ -2,6 +2,7 @@ import type { TableId } from "metabase-types/api";
 
 export interface DataSourceSelectorProps {
   isInitiallyOpen: boolean;
+  isQuerySourceModel: boolean;
```

Comment:
Could you add an inline comment here to what this field means? My assumption is that this means "the query uses a model", but I think I am wrong.

Reply:
It means "is query source a model?", is the starting data source a model. Not just this query use a model at any stage, but if the query uses the model in the first stage.

Reply:
fixed in 2cd19962d4df508d573d255ba3a0e95e8ab373f4